src/Redux/Actions/index.js

```diff
@@ -12,5 +12,9 @@ export default {
   CHANGE_STATE: 'CHANGE_STATE',
   GET_TAGS: 'GET_TAGS',
   AUTO_SAVE: 'AUTO_SAVE',
-  PUBLISH_ARTICLE: 'PUBLISH_ARTICLE'
+  PUBLISH_ARTICLE: 'PUBLISH_ARTICLE',
+  READ_ARTICLE: 'READ_ARTICLE',
```

Comment:
Work on this man @MCFrank16

Reply:
specificity boy, I don't get what you mean by that

Reply:
I need specificity on this boy, I don't get what you mean

Reply:
CHANGE THE WHOLE LOGIC TO GET_ARTICLE INSTEAD OF READ ARTICLE